contracts/Bridge.sol

```diff
@@ -85,14 +89,8 @@ contract Bridge {
         _relayerThreshold = initialRelayerThreshold;
     }
 
-    function getCurrentRelayerThresholdProposal() public view returns (
-        uint256, address[] memory, address[] memory, RelayerThresholdProposalStatus) {
-        return (
-            _currentRelayerThresholdProposal._proposedValue,
-            _currentRelayerThresholdProposal._yesVotes,
-            _currentRelayerThresholdProposal._noVotes,
-            _currentRelayerThresholdProposal._status
-        );
+    function getCurrentRelayerThresholdProposal() public view returns (RelayerThresholdProposal memory) {
```

Comment:
isn't `_currentRelayerThresholdProposal` a public param that can be queried?

Reply:
Yes, but from what you've said in the past about the Go bindings, explicitly returning a struct made it easier on the Go - such as [here](https://github.com/ChainSafe/chainbridge-solidity/blob/1dc1918db4a057bd978bcfb4c1feb76a527e0492/contracts/handlers/ERC20Handler.sol#L33) when returning a deposit record or [here](https://github.com/ChainSafe/chainbridge-solidity/blob/1dc1918db4a057bd978bcfb4c1feb76a527e0492/contracts/handlers/ERC20Handler.sol#L33) when returning a deposit proposal

Reply:
ahhh I see what you're saying, whenever we're returning > 1 argument a struct is best case. I think this might not be needed though?